deploy/kubernetes/helm/che/templates/resource-monitor-role.yaml

```diff
@@ -0,0 +1,34 @@
+#
+# Copyright (c) 2012-2019 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+
+{{- if not (contains "<" .Values.global.cheWorkspacesNamespace) }}
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: resource-monitor
+  namespace: {{ .Values.global.cheWorkspacesNamespace }}
+  labels:
+    app: che
+    component: che
+rules:
+- apiGroups:
+  - ""
+  attributeRestrictions: null
+  resources:
+  - pods
+  verbs:
+  - list
+- apiGroups:
+  - "metrics.k8s.io"
+  attributeRestrictions: null
```

Comment:
Do we really need null to add to the spec ?

Reply:
same commened above.